suite-common/connect-popup/src/connectPopupThunks.ts

```diff
@@ -57,11 +57,11 @@ export const connectPopupCallThunkInner = createThunk<
                 dispatch(
                     connectPopupActions.initiateCall({
                         state: 'call-error',
```

Comment:
this does not have any UI yet, or does it? something similar to https://github.com/trezor/trezor-suite/blob/develop/suite-native/module-connect-popup/src/screens/ConnectPopupScreen.tsx#L99

Reply:
Not yet on desktop. Also the `push_tx` restriction is only needed on mobile due to the callback